src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -708,3 +709,34 @@ def list_nfs_cluster(self):
         except Exception as e:
             log.exception("Failed to list NFS Cluster")
             return getattr(e, 'errno', -1), "", str(e)
+
+    def show_nfs_cluster_info(self, cluster_id):
+        try:
+            self._set_cluster_id(cluster_id)
+            completion = self.mgr.list_daemons(daemon_type='nfs')
+            self.mgr._orchestrator_wait([completion])
+            orchestrator.raise_if_exception(completion)
+            # Here completion.result is a list DaemonDescription objects
+            for cluster in completion.result:
+                if self.cluster_id == cluster.service_id():
+                    """
+                    getaddrinfo sample output: [(<AddressFamily.AF_INET: 2>,
+                    <SocketKind.SOCK_STREAM: 1>, 6, 'xyz', ('172.217.166.98',2049)),
+                    (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '',
+                    ('2404:6800:4009:80d::200e', 2049, 0, 0))]
+                    """
+                    try:
+                        host_ip = {
+                                "hostname": cluster.hostname,
+                                'IP': list(set([ip[4][0] for ip in socket.getaddrinfo(
+                                    cluster.hostname, 2049, flags=socket.AI_CANONNAME,
+                                    type=socket.SOCK_STREAM)])),
+                                'Port': 2049  # Default ganesha port
+                                }
+                        return (0, json.dumps(host_ip, indent=4), '')
```

Comment:
What if there are multiple NFS daemons? Maybe add the daemon_id as a key?
```json
{       
    "ganesha-foo.host1": {
        "hostname": "host1",
        "IP": [
            "192.168.0.1"
        ],  
        "Port": 2049
    },  
    "ganesha-foo.host3": {
        "hostname": "host3",
        "IP": [
            "192.168.0.2"
        ],  
        "Port": 2049
    }   
}
```

Reply:
It's a good idea, done.